scapy/contrib/mqtt.py

```diff
@@ -0,0 +1,251 @@
+## This file is part of Scapy
+## See http://www.secdev.org/projects/scapy for more informations
+## Copyright (C) Santiago Hernandez Ramos <shramos@protonmail.com>
+## This program is published under GPLv2 license
+
+
+from scapy.packet import *
+from scapy.fields import *
+from scapy.layers.inet import TCP
+
+
+# CUSTOM FIELDS
+
+class VariableFieldLenField(FieldLenField):
```

Comment:
When you add code from e.g. stackoverflow, please be kind enough to say so and add a comment with a link to this code (https://stackoverflow.com/a/43722441/3223422 here).